sparse/tests/test_coo.py

```diff
@@ -1606,3 +1606,10 @@ def test_coo_valerr():
 def test_random_idx_dtype():
     with pytest.raises(ValueError):
         sparse.random((300,), density=0.1, format="coo", idx_dtype=np.int8)
+
+
+def test_html_for_size_zero():
+    try:
+        html_table(sparse.COO.from_numpy(np.array(())))
```

Comment:
The try catch is unnecessary, the test fails on an exception.

Also, we should check the result of density and storage ratio, that's where the original bug was.